amundsen_application/static/js/components/Tags/TagInput/index.tsx

```diff
@@ -303,4 +305,4 @@ export const mapDispatchToProps = (dispatch: any) => {
   return bindActionCreators({ getAllTags, updateTags } , dispatch);
 };
 
-export default connect<StateFromProps, DispatchFromProps, ComponentProps>(mapStateToProps, mapDispatchToProps)(TagInput);
+export default connect<StateFromProps, DispatchFromProps>(mapStateToProps, mapDispatchToProps)(TagInput);
```

Comment:
Making a note to add tests for this component as part of the final review for the whole branch. Not blocking merge on this though since it's pre-existing tech-debt.